README.md

```diff
@@ -212,6 +212,7 @@ Tools for interacting with the secret network and for writing Secret Contracts
 * [Secret Py](https://pypi.org/project/secret-sdk/) - Python library for interacting with the network
 * [secret.js](https://github.com/scrtlabs/secret.js) - JavaScript library for interacting with the network
     ([examples](https://github.com/scrtlabs/SecretJS-Templates))
+* [secretk](https://github.com/eqoty-labs/secretk) - Kotlin Multipltform (JVM, Android, iOS, macOS, JS) library for interacting with the network
```

Comment:
```suggestion
* [secretk](https://github.com/eqoty-labs/secretk) - Kotlin Multi-platform (JVM, Android, iOS, macOS, JS) library for interacting with the network
```

Reply:
That's not normally how multiplatform is spelt:
https://kotlinlang.org/docs/multiplatform.html

But, up to you. Just lmk to confirm.

Reply:
The dash isn't important to me, but you were missing an `a` between `pl` and `tform`

Reply:
On another look, I'd also like it to be one line higher, before secretjs, after secretpy